### DISCUSSION TITLE
【求助】我的电脑好像遭受空洞侵蚀了

### DISCUSSION THREAD

**Discussion by March-777th** [share121/inter-knot#1766] [其他]
![三月挠头](https://github.com/user-attachments/assets/8d7a4605-eee2-4594-94fc-bb0d99982c40)

## 一些帖子下面的评论重复了，各位有出现这种问题吗
![image](https://github.com/user-attachments/assets/c11b92f8-c61d-473d-965a-ecd88410212a)
![image](https://github.com/user-attachments/assets/7added0b-61c1-4433-931f-ac75013af3ae)
![image](https://github.com/user-attachments/assets/5f64c62d-3ee1-4ab7-bad1-1ee9c52661e1)

**Comment by capacitor1** [comment]
确实，应该发到issue

**Comment by SednaMi** [comment]
找写回复被骗了2次
![image](https://github.com/user-attachments/assets/976f7c14-5e69-4bb9-a667-30d923e899d5)

**Comment by user1-cloud** [comment]
+1
